### DISCUSSION TITLE
2021-23: 向用户低头的一天

### DISCUSSION THREAD

**Discussion by Xuanwo** [Xuanwo/blog#216] [Announcements]
https://xuanwo.io/reports/2021-23/ 

 An infrastructure engineer, focused on distributed storage system

**Comment by Sasasu** [comment]
https://github.com/WebKit/WebKit/blob/8afe31a018b11741abdf9b4d5bb973d7c1d9ff05/Source/WebCore/dom/ScriptedAnimationController.cpp#L153-L155

然而返回的时间固定是 XX.1 ms

**Comment by Xuanwo** [comment]
不懂前端（:
